tests/TestCase/Utility/HashTest.php

```diff
@@ -1476,6 +1476,17 @@ public function testInsertMulti()
             4 => ['Item' => ['id' => 5, 'title' => 'fifth']],
         ];
         $this->assertEquals($expected, $result);
+
+        $data[3]['testable'] = true;
+        $result = Hash::insert($data, '{n}[testable].Item[id=/\b2|\b4/].test', 2);
```

Comment:
Fair point, we should also implement these additions to remove(). I think that is the only other method that needs changing at this point.